### ISSUE TITLE
Reverse image search button doesn't work

### ISSUE THREAD

**Issue by pmk01** [DemocracyClub/yournextrepresentative#1987]
On candidate photo review. But then I never use it and no-one has raised it so maybe we don't need it?

**Comment by VirginiaDooley** [comment]
I've looked into this and found the search by url link that we could add, but I'm not sure how useful this feature is anymore. For example: 

The google image search by url: https://lens.google.com/uploadbyurl?url=

Combined with...

Our image url: https://s3.eu-west-2.amazonaws.com/static-candidates.democracyclub.org.uk/media/cache/b1/a8/b1a8c8cbef48eb93f01d220cf61bfc14.jpg

Yields [this result](https://lens.google.com/search?p=ARADZa7rfGqdCQUj3MXFiuukqNrR31c-T70EYSa914cQYpkQ0wngYxU0bT2WomQo4n7iCtOIHWZ001w1oXewZYl7TzltEPpz2iXElOTC18mzWyyDfmI62IhFF8UAXmbALEBriqhGYqtqR2T5On1Glw6Ej2y-D9RrVPzgJ40BmH8D3TrYGl429B2AtTvm1tsPk-9fOddigstRc4xRTpsbw7vqxmyZx5TrHlkYdfgFwAJwypi8fD3FoQKx7VHzvNYzZj0I_XO5YjRZAseVTqgbjVvmCYl1nUBjrLX5YScZ#lns=W251bGwsbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsIkVrY0tKRFppTmprek5EVmxMV0ZsTTJNdE5EQTJPQzA0T0dZeUxXVmpOek0xT0RRMk1HUTVZUklmWjNodmJHdHNWbHB4TVd0VFVVUjFWV2RKVkUxaWJEWm5XbEEwYkdWNFp3PT0iLG51bGwsbnVsbCxudWxsLDEsbnVsbCxbbnVsbCxudWxsLFs1NzEsMCwxMDAwMDAsOTc0MjldLG51bGwsNV0sbnVsbF0=)

I also checked the upload functionality of google image search and it yielded the same result. 

I wonder if this tool has become too commercial to be useful to us?
